api/pkg/transformer/pipeline/create_table_op.go

```diff
@@ -146,7 +146,7 @@ func toRawTable(jsonObj interface{}, addRowNumber bool) (map[string]interface{},
 	for idx, j := range jsonArray {
 		node, ok := j.(map[string]interface{})
 		if !ok {
-			return nil, errors.New("not an array of JSON object")
+			return nil, mErrors.NewInvalidInputError("not an array of JSON object")
```

Comment:
With the addition of UPI, is this error message relevant? or should we change it to `not an array of object` ?